Util.lua

```diff
@@ -10,8 +10,8 @@ function Util.formatForcesText(
   currentPullFormat, customCurrentPullFormat,
   pullCount, currentCount, totalCount, completedTime
 )
-  local currentPercent = (currentCount / totalCount) * 100
-  if currentPercent > 100.0 then currentPercent = 100.0 end
+  local currentPercent = math.min((currentCount / totalCount) * 100, 100.0)
+  local currentPulledPercent = (currentCount / totalCount) * 100
```

Comment:
Hmm, so if I understand this correctly, this will show above 100 if the current pull exceeds 100 percent? I'm not sure how I feel about that behavior, since that's basically an additional "hidden feature" of this format string, for people that want to show how much they went over 100...

Maybe it would be better to add a general flag that allows showing when you're over 100 for any format string, and keep this the same as `currentPercent`?

That would simplify this line to `local currentPulledPercent = currentPercent`, and below you could just do

```lua
currentPulledPercent = math.min(currentPercent + pulledPercent, 100.0)
```

Then, if you want to add a flag for showing 100+, you can pull the `math.min` out into a function that would be called something like `clampPercent`, which can check if the flag is set and return the value unchanged or clamp it.

Reply:
Yea, I was (or am) quite unsure about that as well. I could actually just revert to one of my older commits (from git reflog) where the code was more or less exactly like that (if the current pull would bring the percentage above 100, it is floored).
Also thought about adding an option for that (but didn't want to bring the discussion into this MR) so I might just go ahead and implement that right away.
Would an option toggle like "Limit forces count shown to 100%" be fine for you? I'm open for better naming options! :)

Reply:
Please have another look at the updated code and feel free to request any changes.
Thanks for your input!